.github/workflows/rpm.yml

```diff
@@ -0,0 +1,37 @@
+name: Build_rpm
+on:
+  push: 
+    tags:
+      - 'ec2rl-v*'
+
+jobs:
+  Build_rpm:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v2
+
+        - name: Make rpm
+          run: |
+            make rpm
+            PATHNAME=$(ls rpmbuild/*.rpm | head -n 1)
```

Comment:
Nit: this is probably ok in this narrow context, but as a best practice I recommend using find over ls in scripts.

https://github.com/koalaman/shellcheck/wiki/SC2012

Reply:
I will keep this in mind if future changes are needed, thank you!